oss.krtirtho.Spotube.yml

```diff
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: oss.krtirtho.Spotube
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+rename-desktop-file: spotube.desktop
+rename-appdata-file: spotube.appdata.xml
+command: spotube
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --filesystem=~/.local/share/oss.krtirtho.spotube:create
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: spotube
+    buildsystem: simple
+    build-commands:
+      - install -dm755 /app/bin /app/spotube /app/share/appdata /app/share/applications /app/share/icons/hicolor/128x128/apps/
+      - sed -i 's|/usr/bin/spotube|spotube %u|' spotube/spotube.desktop
+      - sed -i 's|/usr/share/icons/spotube/spotube-logo|oss.krtirtho.Spotube|' spotube/spotube.desktop
+      - cp -R spotube/ /app
+      - cp spotube/spotube-logo.png /app/share/icons/hicolor/128x128/apps/oss.krtirtho.Spotube.png
+      - cp spotube/spotube.desktop /app/share/applications/spotube.desktop
+      - cp oss.krtirtho.Spotube.appdata.xml /app/share/appdata/spotube.appdata.xml
+      - ln -s /app/spotube/spotube /app/bin/spotube
+    sources:
+      - type: archive
+        url: https://github.com/KRTirtho/spotube/releases/download/v1.1.0/Spotube-linux-x86_64.tar.xz
```

Comment:
since this is single architecture, you should add a `flathub.json` to make it so https://github.com/flathub/flathub/wiki/App-Maintenance#flathubjson

Or better support `aarch64`.